code/modules/reagents/reagent_containers/hypospray.dm

```diff
@@ -54,14 +54,23 @@
 	list_reagents = list("omnizine" = 30)
 	resistance_flags = INDESTRUCTIBLE | LAVA_PROOF | FIRE_PROOF | ACID_PROOF
 
+/obj/item/reagent_containers/hypospray/emergency_combat
+	name = "emergency combat stabilizer" //Used for the new combat medical kit.
+	desc = "A small modified air-needle auto injector, used by support operatives to quickly stabilize their allies in critical condition."
+	amount_per_transfer_from_this = 5
+	icon_state = "hypo"
+	volume = 35
+	ignore_flags = 1 // So they can heal their comrades.
+	list_reagents = list("inacusiate" = 5, "stimulants" = 15, "atropine" = 15)

```

Comment:
Yes, the helmet is flashbang proof, however your hearing still can be damaged, no matter the mode of the hardsuit. Inacusiate is just there, to fix your ears, when you stand too close to an explosion .
You also don't need to stabilize body temperatures, when that's not even a feature, that TG has. Your body will never drop in heat, if you lose blood or are in a critical condition.

Reply:
You never know, space is still cold and the syndie hardsuit mode can't be switched by the unconscious or third party. The flashbang damage seems like an overlook, rather than an intended feature.

Reply:
If that's the case, the patient will drastically die from the huge brute damage, that space already provides. Leporazine does not give protection against space, it only stops the 0.5burn damage per tick, you received from the coldness.